VikingQuiz.Api/Utilities/AzureBlobService.cs

```diff
@@ -0,0 +1,109 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Configuration;
+using Microsoft.IdentityModel.Tokens;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Blob;
+using Microsoft.WindowsAzure.Storage.Shared.Protocol;
+using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.IO;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+using VikingQuiz.Api.Models;
+
+namespace VikingQuiz.Api.Utilities
+{
+    public class AzureBlobService
+    { // should be changed to a SINGLETON if posibile
```

Comment:
Please remove this comment and put a task in Jira instead. You can call it "Investigate if AzureBlobService can be made a singleton" or something similar